scripting/triggers/bgeetriggers.htm

```diff
@@ -5,1013 +5,1013 @@
 	<div class="title_main"> BG(2)EE Script Triggers </div>
 	<br />
 
-	<div class="triggerHeader"><a name="0x0001">0x0001 Acquired(S:ResRef*)</a></div>
+	<div class="triggerHeader"><b><a name="0x0001">0x0001 Acquired(S:ResRef*)</a></b></div>
```

Comment:
This is silly, I suggest you revert this whole file and just update the style, adding `font-weight: bold` to the triggerHeader css class.

Reply:
> This is silly, I suggest you revert this whole file and just update the style, adding `font-weight: bold` to the triggerHeader css class.

Done (they're now treated as `.actionHeader`, i.e.: `{font-size: 14px; font-weight: bold;}`)

Anyway, the next step here is to turn all of this into `yml` files, just like script actions? That'll take a huge amount of time...

Reply:
That's more of a task for some scripting and regular expressions, as it would be a bit silly to do by hand.

Reply:
> That's more of a task for some scripting and regular expressions, as it would be a bit silly to do by hand.

That's certainly true. It should be relatively simple to write a WeiDU mod that generates a `yml` file for each script trigger... But then you still need to copy-paste descriptions manually...?

Anyway, @burner1024 knows better than me, since he's already did script actions...

Reply:
Doing it via WeiDU sounds like first-class masochism! 😮

Reply:
Converting these to yaml would likely be not very hard. However:
1. I would propose to keep data changes and format changes separate.
2. Converting requires a defined format of the file, and as far as I remember, there were some unresolved items in discussion about that.
3. It will also require corresponding changes in html templates.

@4Luke4 if you take care of 2) and 3) (preferably after merging this pull), I'll give mass conversion a try.